index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="desktop-style.css" />
+    <link rel="stylesheet" href="mobile-style.css" />
+    <title>Capstone</title>
+  </head>
+  <body>
+    <header class="black-menu hide">
+      <ul class="black-menu hide">
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/facebook.png"
+              alt="facebook-logo"
+              height="20"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/twitter.png"
+              alt="twitter-logo"
+              height="20"
+          /></a>
+        </li>
+        <li>
+          <a href="#">English</a>
+        </li>
+        <li>
+          <a href="#">My Page</a>
+        </li>
+        <li>
+          <a href="#">Logout</a>
+        </li>
+      </ul>
+    </header>
```

Comment:
- [ ] <b>REQUIRED:</b> As mentioned in the previous review, make sure that your toolbar interface looks exactly like the toolbar in mockup design.

| Original      | Yours |
  | ----------- | ----------- |
| ![scrnli_9_19_2021_4-18-18 AM](https://user-images.githubusercontent.com/45217364/133913391-9805e276-3e28-4856-af55-ae425b9ede37.png) |  ![scrnli_9_19_2021_4-23-18 AM](https://user-images.githubusercontent.com/45217364/133913394-d80c71e5-015b-4957-a2d0-8e554383452c.png) |

   - All links must be vertically centered.

Reply:
> * [ ]  REQUIRED: As mentioned in the previous review, make sure that your toolbar interface looks exactly like the toolbar in mockup design.
> 
> Original	Yours
> ![scrnli_9_19_2021_4-18-18 AM](https://user-images.githubusercontent.com/45217364/133913391-9805e276-3e28-4856-af55-ae425b9ede37.png)	![scrnli_9_19_2021_4-23-18 AM](https://user-images.githubusercontent.com/45217364/133913394-d80c71e5-015b-4957-a2d0-8e554383452c.png)
> * All links must be vertically centered.

> This section of your project is actually the first part and most important part of your project, since users would get to spend more time here, it would be great if you try to mimic the original as much as possible. 🙏